std/internal/math/gammafunction.d

```diff
@@ -531,8 +543,17 @@ unittest {
 
 
 private {
-enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
-enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min*real.epsilon) = log(smallest denormal)
+static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended) {
+    enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
+    enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
+}
+else static if (floatTraits!(real).realFormat == RealFormat.ieeeDouble) {
+    enum real MAXLOG = 0x1.62e430p+9L;  // log(real.max)
+    enum real MINLOG = -0x1.743854p+9L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
```

Comment:
Floating hexadecimal approximate constants for unittests can be computed with Wolfram Alpha.
Example: [hexadecimal log(2^52)](http://www.wolframalpha.com/input/?i=hexadecimal+log%282%5E52%29).
Also you can press *More digits*.

For this example you need exact value of `double.max` and `double.min_normal*real.epsilon`. Probably you can print them in hexadecimal form, see `std.format` for more details.

Please add note about Wolfram into comments and the date when you would computed this values using Wolfram.